sunpy/net/solarnet/solarnet.py

```diff
@@ -0,0 +1,181 @@
+    <BLANKLINE>
+    <BLANKLINE>
+    """
+
+    @property
+    def info_url(self):
+        return 'https://solarnet2.oma.be'
+
+
+
+    def search(self, *query):
+        """
+        Searches the SVO for datasets matching the query.
+
+        Parameters
+        ----------
+        *query : `~sunpy.net.attrs`
+            Attributes to search for datasets.
+
+        Returns
+        -------
+        `~sunpy.net.base_client.QueryResponseTable`
+            A table containing the search results.
+        """
+        results = []
+        query = and_(*query)
+        block = walker.create(query)[0]
+
+        if "datasets" in block:
+            url = _BASE_URL.format(block["datasets"])
+            block.pop("datasets")
+
+        try:
+            req = urllib.request.urlopen(url + '?' + urllib.parse.urlencode(block))
+            data = json.loads(req.read().decode())["objects"]
+        except Exception as e:
+            raise Exception(f"Failed to fetch data from {url}: {str(e)}")
+
+        hide_keys = ["Fits_header", "Data_location", "Resource_uri", "url", "File_raw", "File_tar", "File_tmr"]
+
+            # just to avoid empty tags
+        hide_keys.append("Tags") if "tag" not in block else hide_keys
+
+        for i in data:
+                    # Filter out keys that should be hidden or not displayed
+                filtered_data = {key.capitalize(): value for key, value in i.items()}
+
+                    # Add specific fields from data_location
+                filtered_data.update({
+                        "F_size": i["data_location"]["file_size"],
+                        "url": i["data_location"]["file_url"],
+                    })
+
+                results.append(filtered_data)
+
+
+
+        qrt = QueryResponseTable(results, client=self)
+        qrt.hide_keys = hide_keys
+        return qrt
+
+    @staticmethod
+    def load_solarnet_values():
+        """
+        Load Solarnet dataset values from a local JSON file.
+
+        Returns
+        -------
+        attrs : `dict`
+            Dictionary of Solarnet dataset values.
+        """
+        data_sets = Path(__file__).parent / "data" / "datasets.json"
+        tags = Path(__file__).parent / "data" / "tags.json"
+
+        with data_sets.open() as data_values:
+            data = json.load(data_values)
+
+        with tags.open() as tags_values:
+            tags = json.load(tags_values)
+
+        data_set = {Dataset: list(data.items())}
+        target = {Target: [("AR", "Active Region"),("CH","Coronal Hole"),("FS","Flare"),("QR","Quiet Region")]}
+        tags = {Tags:list(tags.items())}
+
+        attrs = data_set | target | tags
+        return attrs
+
+    def create_parse_solarnet_values():
```

Comment:
This method has no test coverage.

Reply:
Do we test any of the method which create the json files?